tool/dev.dart

```diff
@@ -31,6 +31,7 @@ Future<Null> main(List<String> args) async {
   ];
   config.copyLicense.directories = directories;
   config.format.directories = directories;
+  config.test.platforms = ['content-shell', 'vm'];
```

Comment:
If you switch to the individual w_common entry points above, I believe you'll be able to run these tests on the VM instead of in the browser